napari/utils/interactions.py

```diff
@@ -236,7 +236,7 @@ def hello_world(layer, event):
 }
 
 
-joinchar = '-'
+joinchar = '+'
```

Comment:
To be consistent with Qt shortcut representation

Reply:
We use `-` in the definition of shortcuts almost everywhere, and I don't think endusers are exposed to QT internals. As this is used to display shortcuts I believe, I want to make sure it's consistent UI wise.